README.md

```diff
@@ -404,6 +404,56 @@ uwsgi --http 127.0.0.1:8000 --wsgi-file myapp.py --callable app
 
 Visit http://localhost:8000/metrics to see the metrics
 
+#### FastAPI + Gunicorn
+
+To use Prometheus with [FastAPI](https://fastapi.tiangolo.com/) and [Gunicorn](https://gunicorn.org/) we need to serve metrics through a Prometheus ASGI application.
+
+Save the snippet below in a `myapp.py` file
+
+```python
+from fastapi import FastAPI
+from prometheus_client import make_asgi_app
+
+# Create app
+app = FastAPI(debug=False)
+
+# Add prometheus asgi middleware to route /metrics requests
+metrics_app = make_asgi_app()
+app.mount("/metrics", metrics_app)
+```
+
+For Multiprocessing support, use this modified code snippet. Full multiprocessing intstructions are provided [here](https://github.com/prometheus/client_python#multiprocess-mode-eg-gunicorn).
+
+```python
+from fastapi import FastAPI
+from prometheus_client import make_asgi_app
+
+app = FastAPI(debug=False)
+
+# Using multiprocess collector for registry
+def make_metrics_app():
+    registry = CollectorRegistry()
+    multiprocess.MultiProcessCollector(registry)
+    return prometheus_client.make_asgi_app(registry=registry)
```

Comment:
@ashirviskas should be based on the [import](https://github.com/prometheus/client_python/pull/812/commits/60ced23a24abbd2694e405a128db37afa8bfd52d#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R429)
```python
return make_asgi_app(registry=registry)
```

Reply:
I might be late, but I just fixed it